patterns/2-structured/feature-owners.md

```diff
@@ -0,0 +1,75 @@
+## Title
+
+Feature Owners
+
+## Patlet
+
+This pattern is for scaling InnerSource project contributions aligned with business initiatives and for aspirant employees who are looking for stretch assignments, more accountability, develop their soft skills and be successful at work.
+
+## Problem
+
+Contributions in InnerSource projects mostly are all volunteer based. Below are the challenges we see in day-to-day life:
+
+- How to increase more contributions in the project.
```

Comment:
```suggestion
- How to increase contributions to the project.
```

`increase` and `more` said the same thing. simplifying.

Reply:
Sure, thanks.

Reply:
FYI if you have not seen this yet, you should see the option to commit suggested changes directly here in the GitHub UI. For small changes this is often the easiest way.